pandas/core/frame.py

```diff
@@ -2901,6 +2902,93 @@ def to_parquet(
             **kwargs,
         )
 
+    def to_orc(
+        self,
+        path: FilePath | WriteBuffer[bytes] | None = None,
+        *,
+        engine: Literal["pyarrow"] = "pyarrow",
+        index: bool | None = None,
+        **kwargs,
+    ) -> bytes | None:
+        """
+        Write a DataFrame to the ORC format.
+
+        .. versionadded:: 1.5.0
+
+        Parameters
+        ----------
+        path : str, file-like object or None, default None
+            If a string, it will be used as Root Directory path
+            when writing a partitioned dataset. By file-like object,
+            we refer to objects with a write() method, such as a file handle
+            (e.g. via builtin open function). If path is None,
+            a bytes object is returned.
+        engine : {{'pyarrow'}}, default 'pyarrow'
+            ORC library to use, or library it self, checked with 'pyarrow' name
+            and version >= 7.0.0. Raises ValueError if it is anything but
```

Comment:
IMO `Raises ValueError if it is anything but...` is redundant with the raises section below so I think this can be removed.

Reply:
Fixed.